runtime/src/execute_cost_table.rs

```diff
@@ -17,10 +19,65 @@ const OCCURRENCES_WEIGHT: i64 = 100;
 
 const DEFAULT_CAPACITY: usize = 1024;
 
-#[derive(AbiExample, Debug)]
+// The EMA_ALPHA represents the degree of weighting decrease in EMA,
+// a constant smoothing factor between 0 and 1. A higher alpha
+// discounts older observations faster.
+// Estimate it to 0.01 by `2/(N+1)` where N is 200 samples
+const EMA_ALPHA: u16 = 10;
+const EMA_SCALE: u16 = 1000;
```

Comment:
representing `0.01` as `10 / 1000`, so  math can be done with integer, with 3 decimals precision.

Reply:
@behzadnouri this is the same EMA impl as last, but to do math with integer. Basically I inflates all number by `1000` then math with saturating_* as integers, at the end div result by `1000`.  That reduces precision from float to 3 decimal point, most tests are not complaining. Do you have concern with this approach? thanks.

Reply:
Not really sure if there is a scenario that this will behave weirdly.
What is the motivation for doing so?

Reply:
mainly to the concern of casting from `u64` (input data type) to `f64` (intermediate type for math, partly because alpha = 0.01), then again to `u64` (result type). And unchecked math ops for `f64`. 

In my previous life coding for trading system, it's common to represent floats as integers with decimal points, much like I did here, to make math safer and stable.  On top of reduced precision, this literally reduces max value by `1000` times from `u64::MAX`. But I think that might be OK with our use case. I also curious if there is better way in Rust to have situation like this.

Reply:
My first intuition is do not bother unless there is really a necessity.
Another alternative is `FixedI64` below which has `checked_...` math:
https://docs.rs/fixed/latest/x86_64-pc-windows-msvc/fixed/struct.FixedI64.html

Reply:
+1